toolset/setup/linux/frameworks/cppcms-cppdb.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+fw_installed cppcms-cppdb && return 0
+fw_depends cppcms
+
+#HOW THAT GOES
+#http://cppcms.com/sql/cppdb/build.html
+
+VERSION=0.3.1
+BACKNAME=cppdb
+CPPCMS_HOME=$IROOT/$BACKNAME-$VERSION
+
+sudo apt-get update -qq
+#unixodbc-dev libsqlite3-dev
+sudo apt-get install -y cmake libmysqld-dev libpq-dev
+
+fw_get -o $BACKNAME-$VERSION.tar.bz2 https://download.sourceforge.net/project/cppcms/$BACKNAME/$VERSION/$BACKNAME-$VERSION.tar.bz2
+fw_untar $BACKNAME-$VERSION.tar.bz2
+
+cd $BACKNAME-$VERSION
+rm -rf build
+mkdir build
+cd build
+cmake ..
+
+make
+#make test
+sudo make install
```

Comment:
Please take some time to try and get this installed without `sudo` and in the `$IROOT` directory.